attest/verifier/config/Cargo.toml

```
@@ -9,8 +9,10 @@ readme = "README.md"
 [dependencies]
 displaydoc = { version = "0.2", default-features = false }
 hex = { version = "0.4.3", default-features = false, features = ["alloc", "serde"] }
+mc-attestation-verifier = "0.2.0"
+mc-sgx-core-types = "0.7.2"
 serde = { version = "1.0", default-features = false, features = ["derive"] }
-serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
+serde_json = { version = "1.0.103", default-features = false, features = ["alloc"] }
```

:question: Do you know what caused this to change to include the patch version? Curious if it failed to build or similar.

I think we for the attestation and sgx repo we need to remove the patch version as it's going to force tighter constraints then necessary.

```suggestion
serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
```